### ISSUE TITLE
Feature Request: Support GriefPrevention Selections

### ISSUE THREAD

**Issue by TomLewis** [KennyTV/WorldEditSUI#16]
I see that GriefDefender has already been requested: https://github.com/kennytv/WorldEditSUI/issues/11
I use GriefPrevention, so it would be fantastic to support both! They use the samish API!

**Comment by kennytv** [comment]
I haven't used either of these mods, can you elaborate on what exactly/when you want something to be visualized?

**Comment by TomLewis** [comment]
Yeah sure! GriefPrevention lets players create claims for their bases, they use a tool (Wooden stick by default) to right click on alnd and it "highlights" the land by showing the corners of their claim in Gold/glowstone. The problem with this is, its hard to see!

Heres a guide I wrote on my website with images for easy understanding https://piratemc.com/guides/claims/

Players can also have sub-claims which let them sub divide their claims to allow access to only certain parts of their claims, this could be shown in another type of particle.

Other than that, theres admin claims that just have differnt blocks to show they are an admin claim.

And the biggest and best tool for admins is you can fly and shift+right click and itll show you all claims within a radious of you to quickly see where claims are, this is the key reason I would personally love it, to make visualising many claims around the world by flying around as particles would be much more visual!
